praw/models/reddit/redditor.py

```diff
@@ -44,6 +44,8 @@ class Redditor(MessageableMixin, RedditorListingMixin, FullnameMixin, RedditBase
     ``comment_karma``                    The comment karma for the Redditor.
     ``comments``                         Provide an instance of :class:`.SubListing`
                                          for comment access.
+    ``submissions``                      Provide an instance of :class:`.SubListing`
```

Comment:
Thanks for taking the time to improve PRAW's documentation.

I actually would argue that neither `comments` nor `submissions` should be in this list because they are not dynamic attributes, but rather attributes that are statically part of the class, and documented already here:

https://praw.readthedocs.io/en/latest/code_overview/models/redditor.html#praw.models.Redditor.comments
https://praw.readthedocs.io/en/latest/code_overview/models/redditor.html#praw.models.Redditor.submissions

If you would like to update your pull request to remove `comments` from this list, then we can accept this PR. Otherwise, I think we should close it.

Reply:
Actually they are dynamic, they are not an attribute if is_suspended

Reply:
I don't quite follow. The attributes seem to still exist:

```
In [3]: suspended = reddit.redditor(SUSPENDED_USER")

In [4]: suspended.is_suspended
Out[4]: True

In [5]: suspended.comments
Out[5]: <praw.models.listing.mixins.redditor.SubListing at 0x103d1e100>

In [6]: suspended.submissions
Out[6]: <praw.models.listing.mixins.redditor.SubListing at 0x103d32460>
```

Reply:
Oh that is surprising, because the note says: "Suspended/banned accounts will only return the name and is_suspended attributes."

Do they also exist if the user has no comments or submissions?

Also, they do not exist if the user is not found.

Reply:
> Oh that is surprising, because the note says: "Suspended/banned accounts will only return the name and is_suspended attributes."

I think that specifically refers to the real dynamic attributes that come back from Reddit. Whereas, what I was explaining before, the properties that we add will always be there, hence why they are documented elsewhere.

> Also, they do not exist if the user is not found.

Yes, that should raise an exception.

Reply:
Oh, huh, I see the issue, but shouldn't ```name``` also be removed then?